shipyard-controller/main.go

```diff
@@ -124,6 +131,8 @@ func main() {
 	shipyardController.AddSequenceFinishedHook(sequenceStateMaterializedView)
 
 	engine.Static("/swagger-ui", "./swagger-ui")
+
+	go keptnapi.RunHealthEndpoint("10998")
```

Comment:
I don't understand, why do we have a `healthController` up there, but also run the health endpoint on a separate port?

Reply:
forgeot to delete this line, will do